### DISCUSSION TITLE
in multiple Rule Attribute, only apply the first attribute near property.!!

### DISCUSSION THREAD

**Discussion by yacobii** [livewire/livewire#6415] [1. Help]
<input type="text" wire:model.live="title" >

#some Rules...
#[Rule('required', message: 'Please provide a post title')]
#[Rule('min:3', message: 'This title is too short')]
public $title;

only show min Rule message.

**Comment by lucazsvn** [comment]
Same issue!

**Comment by nicolagianelli** [comment]
Try using #[Rule(['required', 'min:3'], message: ['required' => 'Please provide a post title', 'min' => 'This title is too short'], translate: false)],
you have to use "translate: false" because I found a bug where translation is not working when passing an array of messages...

I already created a PR ( #6654 ) to fix this issue, anyway the if you aren't using translation for now it should work 😊

**Reply by lucazsvn** [reply]
Thank you so much!
